lib/hello_world.js

```diff
@@ -1,9 +0,0 @@
-"use strict";
```

Comment:
To be able to find the binary when the `module_path` has versioning variables. Because we just use `lib/binding` [here](https://github.com/mapbox/node-cpp-skel/blob/59d00921ce3751b7a688ee22c250e99c74e9b34a/package.json#L30) and not versioning variables like: https://github.com/mapbox/node-sqlite3/blob/f1456b1cc96e89b82ac37b1896f23ef14aff80b6/package.json#L12 we can safely hardcode this path. Overall the ability to use versioning variables in node-pre-gyp I consider an edge case feature/too much flexibility and should likely be removed from node-pre-gyp in future versions. The motivation/need for versioning varables in the `module_path` is to be able to install multiple binaries for multiple node platforms and have them both running locally. Then you could `nvm use <diff version>` and be able to run your app without re-installing. But again, this is a rare need.